Build System Bug: cmake fails if you don't have g++ installed

Hi all, found this small bug that causes cmake to fail if you don't have g++ installed.

Env: Solaris 11 x86
gcc installed, g++ not.

output of cmake ..
    -- Detecting C compiler ABI info - done
    CMake Error: your CXX compiler: "CMAKE_CXX_COMPILER-NOTFOUND" was not found.   Please set CMAKE_CXX_COMPILER to a valid compiler path or name.
    -- Configuration set to: DebugFast
    -- GMP libs: /usr/lib/libgmp.so /usr/lib/libgmpxx.so

Fix: install g++.  I don't know cmake enough, or I would have attached a patch that fixes this in the build system rather than adding another package to the system.

-->Jake

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,7 +32,7 @@ endif()
 	
 # Project name, this gets prefixed to a bunch of stuff under the hood. No
 # spaces, or anything silly like that please.
-project(IoLanguage)
+project(IoLanguage C)
 
 # Default config when building with gcc variants
 IF(CMAKE_COMPILER_IS_GNUCC OR (CMAKE_C_COMPILER_ID MATCHES "Clang"))

